docs/diagrams/UiClassDiagram.puml

```diff
@@ -15,6 +15,8 @@ Class PersonListPanel
 Class PersonCard
 Class StatusBarFooter
 Class CommandBox
+Class EmailWindow
+Class PieChartWindow
 }
 
```

Comment:
Perhaps could include the buttons (resize, copy email) as part of the UI class diagram